tests/integration/reqs/test_submit_multisigned.py

```diff
@@ -21,15 +23,19 @@
 
 FEE = get_fee(JSON_RPC_CLIENT)
 
-#
 # Set up signer list
 FIRST_SIGNER = Wallet.create()
 SECOND_SIGNER = Wallet.create()
 LIST_SET_TX = sign_and_reliable_submission(
     SignerListSet(
         account=WALLET.classic_address,
-        sequence=WALLET.sequence,
-        last_ledger_sequence=WALLET.sequence + 10,
+        sequence=get_next_valid_seq_number_sync(
+            WALLET.classic_address, JSON_RPC_CLIENT
+        ),
+        last_ledger_sequence=get_next_valid_seq_number_sync(
+            WALLET.classic_address, JSON_RPC_CLIENT
+        )
+        + 10,
```

Comment:
This should be fetching the last ledger that passed, not using the sequence number of the wallet.